src/pricing/calculations/roof.js

```diff
@@ -0,0 +1,26 @@
+import { tiles } from '../content/roof/tiles'
+import { slope } from '../content/roof/slope'
+import { overHang } from '../content/roof/overHang'
+
+function parseDimensions(dimensions) {
```

Comment:
Don't you think this would be better (and notice the comment):
```javascript
const parseDimensions = ({floors, homeSquareFoot, garageSquareFoot}) => {
  return {
    floors: parseInt(floors.result.value, 10),
    homeSquareFoot: parseInt(homeSquareFoot.result.value, 10),
    garageSquareFoot: parseInt(garageSquareFoot.result.value || '1', 10)
    // Shouldn't it's default value be 0 instead of 1? We're adding it to the final
    // square foot, and not multiple it.
    // Maybe:
    //
    // garageSquareFoot: parseInt(garageSquareFoot.result.value || '0', 10)
    //
  }
}
```